### PR TITLE
Create AppProgrammingQuickestStart.md

### PR DESCRIPTION
**Pull request by 2xAA**
Adds a _very_ quick and concise guide to get people up and running.

### CODE REVIEW

**Review by alsuren — APPROVED** [review]
I tested locally and this got me started pretty quickly.

~10/10~ 12/12 would quickstart again.

File: AppProgrammingQuickest-start.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+This shows you how to add code into a file, add it to main menu and run it.
+See https://github.com/emfcamp/tidal-docs/blob/main/AppQuickstart.md for more example code!
+
+1. Plug in your TiDAL in your computer with power off
+2. Turn on your TiDAL
+3. Open https://editor.badge.emfcamp.org/ in Google Chrome
+4. Press "Connect", select your TiDAL (Espressif CDC Device on Windows) and press "Connect"
+5. Press "Programming & Files"
+6. Press "Make folder", second button from the left, and create a folder named `apps`
```
Comment (tomsci) [inline AppProgrammingQuickest-start.md:11]: This step is no longer needed (with v0.5.1 or higher firmware)

Suggest we add a "check your firmware is up to date" step at the beginning?

File: AppProgrammingQuickest-start.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    ```Python
+    from tidal import *
+    from textwindow import TextWindow
+
+    win = TextWindow(bg=BLACK, fg=WHITE)
+    win.cls()
+    win.println("Hello world!")
+    ```
+11. Press "Save file", the rightmost button
+12. Reset your TiDAL by pressing the button in the corner and see your app in the list on your TiDAL
```
Comment (tomsci) [inline AppProgrammingQuickest-start.md:25]: Can we finish with a link to https://github.com/emfcamp/tidal-docs/blob/main/AppQuickstart.md#structure-of-an-app at the end for how to make your code into an actual `App` that plays nice with task switching and such?

**Review by tomsci — APPROVED** [review]
Will merge as-is but if you could take a look at my comments that'd be good!

File: AppProgrammingQuickest-start.md (hunk 2 above)
Comment (2xAA) [reply AppProgrammingQuickest-start.md:25]: I'll create a new PR to fix these soon.
Thanks for merging.

**Pull request merged**